lib/swagger-explorer.ts

```diff
@@ -0,0 +1,114 @@
+import { ExecutionContext, INestApplication, Injectable } from '@nestjs/common';
+import { SwaggerScanner } from '@nestjs/swagger/dist/swagger-scanner';
+
+export interface ISchemaIdContextData {
+  schemaId: string;
+  data: any;
+}
+
+@Injectable()
+export class SwaggerExplorer {
+  public getPathSchemasData(context: ExecutionContext): ISchemaIdContextData[] {
+    const controllerName = context.getClass().name;
+    const handlerName = context.getHandler().name;
+    const req = context.switchToHttp().getRequest();
+
+    return ['body', 'query', 'path:params', 'header:headers'].map(
+      (dataType) => {
+        const [schemaParam, reqParam] = dataType.split(':');
+
+        return {
+          data: req[reqParam || schemaParam],
+          schemaId: `${controllerName}_${handlerName}_${schemaParam}`,
```

Comment:
лучше юзать какие нибудь спец символы, если это возможно

```suggestion
          schemaId: `${controllerName}:${handlerName}:${schemaParam}`,
```

так легче будет потом сплитить, потому что метод вполне может содержать `_` и тогда про сплит можно будет забыть